parts/kubernetesmastercustomdata.yml

```diff
@@ -317,11 +185,19 @@ write_files:
     [Service]
     Restart=always
     ExecStartPre=/bin/mkdir -p /var/lib/kubelet
+    ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
```

Comment:
I found that some of the yaml contents which will be extracted to /etc/kubernetes/manifest, and /etc/kubernetes/add-on are gziped and some are not. I just put all these kind of yamls as gzip content to make it clean. And another problem is that if there are too many content in this file, it will be an error when deploying it to Azure due to the limitation of the customData.